projects/robots/husarion/rosbot/controllers/rosbot/rosbot.c

```diff
@@ -102,10 +102,10 @@ int main(int argc, char *argv[]) {
   wb_compass_enable(compass, TIME_STEP);
 
   /* get a handler to the distance sensors and enable them. */
-  distance_sensors[0] = wb_robot_get_device("rear left distance sensor");
-  distance_sensors[1] = wb_robot_get_device("front left distance sensor");
-  distance_sensors[2] = wb_robot_get_device("front right distance sensor");
-  distance_sensors[3] = wb_robot_get_device("rear right distance sensor");
+  distance_sensors[0] = wb_robot_get_device("fl_range");
+  distance_sensors[1] = wb_robot_get_device("rl_range");
+  distance_sensors[2] = wb_robot_get_device("fr_range");
+  distance_sensors[3] = wb_robot_get_device("rr_range");
```

Comment:
same thing, also I suspect we're introducing a bug here since two different devices are being given the same name (both distance sensors and position sensors). I suggest keep the previous nomenclature for all (motor & devices)

Reply:
I see, but they're not very descriptive (like in the case of calling a Lidar "laser"), especially since in Webots we have other devices with similar names that could create confusion. Is it not possible perhaps to improve the URDF itself with more suitable names? Or maybe to just do the change on the webots side and remap the frames in ROS? I understand the change, but to be honest it doesn't seem very clean. Or at least, it seems like a step back compared to the previous PROTO